spec.md

```diff
@@ -308,7 +308,9 @@ Multi-Valued: No
 Since:        0.1.0
 ```
 
-The `TITLE` header contains the title of the song.
+The `TITLE` header contains the title of the song. It SHOULD NOT include any additional
+information. Details such as recording specification or categorization SHOULD be provided using
```

Comment:
To me this reads as if song creators should add these details unless they have a very good reason not to. This would be a little strong for my taste, I'd prefer to just point out the possibility to use these other headers.

```suggestion
information. Details such as recording specification or categorization can be provided using
```

Reply:
I considered that but couldn't come up with a good reason *not* to add the rendition header. It's trivial to add for the creator, but if missing, sooner or later someone will stumble across the text file without a (valid) audio source and have to tediously piece it together.
What do you think?

Reply:
I see your point and I do agree that there is definitely a benefit in including the additional information. To me the upper-case "SHOULD" (and the spec in general) is something that mainly targets the developers of applications and less song creators. In particular the words SHOULD/MAY/... instruct the developers to implement certain behavior. To me it currently reads as if an application is somehow supposed to guess additional information about a song.

What about something like "song creators are encouraged to include additional information using headers like `#RENDITION`, `#EDITION`, and `#TAGS`"? Maybe that makes it more clear who this is aimed at. This applies to the "SHOULD" in the `#RENDITION` header section as well.

Reply:
You're right! I wrote this for song creators, but they are not the target audience of this document. I'll start from scratch.